api_server.py

```
@@ -228,19 +278,34 @@ def health():
 def root():
     try:
         now = datetime.now()
```

Ruff (0.15.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)